ldms/src/store/darshan/Plugin_darshan_stream_store.man

```diff
@@ -0,0 +1,81 @@
+.\" Manpage for Plugin_darshan_stream_store
+.\" Contact ovis-help@ca.sandia.gov to correct errors or typos.
+.TH main 7 "26 September 2021" "v4" "LDMS Plugin darshan_stream_store man page"
+
+.SH NAME
+Plugin_darshan_stream_store - man page for the LDMS darshan_stream_store plugin
```

Comment:
It seems redundant to say "man page for ..." in the man page itself. Maybe just "LDMS Darshan Stream Store Plugin"